packages/components/src/components/button/_button.scss

```diff
@@ -32,7 +32,7 @@
     + .#{$prefix}--btn--primary.#{$prefix}--btn--disabled,
   .#{$prefix}--btn--tertiary.#{$prefix}--btn--disabled
     + .#{$prefix}--btn--danger.#{$prefix}--btn--disabled {
-    border-left: rem(1px) solid $disabled-03;
+    box-shadow: -#{rem(1px)} 0 0 0 $disabled-03;
```

Comment:
Is it possible to just do `rem(-1px)`?

Reply:
Yes, totally. I don't know why it didn't come to mind 😄